processing/src/test/java/org/apache/druid/segment/filter/BaseFilterTest.java

```diff
@@ -24,8 +24,20 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Iterables;
+import junitparams.converters.Nullable;
```

Comment:
Did you mean `javax.annotation.Nullable`?

Reply:
Heh, oops, gg intellij